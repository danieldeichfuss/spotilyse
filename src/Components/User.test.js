import React from "react";
import User from "./User";
import { shallow, configure } from "enzyme";
import { shallowToJson } from "enzyme-to-json";
import ReactSixteenAdapter from "enzyme-adapter-react-16";

configure({ adapter: new ReactSixteenAdapter() });
const user = {
  name: "Name",
  imgUrl: "img-url"
};
const wrapper = shallow(<User user={user} />);
it("renders page correctly", () => {
  expect(shallowToJson(wrapper)).toMatchSnapshot();
});
