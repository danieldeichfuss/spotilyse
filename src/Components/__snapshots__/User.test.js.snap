// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders page correctly 1`] = `
<div
  className="User"
>
  <img
    alt="User avatar"
    className="User__avatar"
  />
  <p
    className="User__intro"
  >
    Hi 
    Name
    ,
    <br />
    welcome to Spotilyse! Here you can expolore your musical personality and discover your favourite artists.
  </p>
</div>
`;
